spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/domain/SubTrade.java

```diff
@@ -38,7 +38,7 @@
 	@PrimaryKey(keyOrder = 2)
 	String subTradeId;
 
-	@Interleaved
+	@Interleaved(lazy = true)
```

Comment:
How do you verify that this is actually doing something? (i.e. like verify the retrieval is truly lazy versus non-lazy)

Reply:
There are existing unit tests in a previous PR that tested for this on both read and write .

Look at 
- SpannerTemplateTests.lazyFetchChildrenTest
- SpannerMutationFactoryImpl.lazyWriteTest

Reply:
Gotcha, however I was more asking if there is a way to verify it's doing something at the integration test level.

Right now it doesn't seem quite right because existence of `lazy = true` doesn't have any affect on the integration test.

Reply:
Yea, I don't think that's the responsibility of an "integration" test. I think verifying how something is done is up to unit tests, but verifying what is done (being blind to what's going on behind the scenes) is all that integration tests are responsible for.

Reply:
Sounds good.

Reply:
https://github.com/spring-cloud/spring-cloud-gcp/blob/a592c888d8ed06011ad165e4f4eb964e986ed36e/spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/domain/SubTrade.java